frontend/src/app/view-claims/view-claims.component.ts

```diff
@@ -56,8 +54,22 @@ export class ViewClaimsComponent implements OnInit {
       () => {});
   }
 
-  openSnackBar() {
-
-
+  sortClaims(num): void{
```

Comment:
Delete this entire method and instead just filter by status on click.

So, if they clicked status 2....

```ts
this.visibileClaims = this.claims.filter(claim => claim.status === 2)
```

Reply:
I left the method there, but instead it returns a sorted array of the claims. The for loop in the .html goes through the sorted list.